### ISSUE TITLE
Error when enabling plugin

### ISSUE THREAD

**Issue by arlarzelere3** [roxaskeyheart/Chromatics#1]
Unable to enable the plugin.  See attached error

![chromatics error](https://cloud.githubusercontent.com/assets/125488/11484399/cfc1fd3a-977b-11e5-9d25-78b70d5a494b.PNG)

I am attempting to use this with a Logitech G13 and G600.  I don't have Razer Synapse installed.

**Comment by arlarzelere3** [comment]
ACT Log Entry:

****\* 2015-11-30T15:46:04 - Enabling plugin: C:\Users\Alex\Desktop\Chromatics_x64_1.1.2\Chromatics\Chromatics.dll
****\* 2015-11-30T15:46:04 - Plugin Initialization Failed
System.TypeInitializationException: The type initializer for 'Corale.Colore.Razer.NativeMethods' threw an exception. ---> Corale.Colore.ColoreException: Failed to dynamically load Chroma SDK library (Error 126).
   at Corale.Colore.Razer.NativeMethods..cctor()
   --- End of inner exception stack trace ---
   at Corale.Colore.Core.NativeWrapper.Init()
   at Corale.Colore.Core.Chroma.Initialize()
   at Corale.Colore.Core.Chroma.get_Instance()
   at Chromatics.Chromatics.InitializeDevices()
   at Chromatics.Chromatics.InitPlugin(TabPage pluginScreenSpace, Label pluginStatusText)
   at Advanced_Combat_Tracker.FormActMain.pluginPanelEnabledChecked(Object sender, EventArgs e)
   at Advanced_Combat_Tracker.FormActMain.WriteExceptionLog(Exception ex, String MoreInfo)
   at Advanced_Combat_Tracker.FormActMain.pluginPanelEnabledChecked(Object sender, EventArgs e)
   at System.Windows.Forms.CheckBox.OnCheckedChanged(EventArgs e)
   at System.Windows.Forms.CheckBox.set_CheckState(CheckState value)
   at System.Windows.Forms.CheckBox.OnClick(EventArgs e)
   at System.Windows.Forms.CheckBox.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
   at System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG& msg)
   at System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG& msg)
   at System.Windows.Forms.Application.ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr dwComponentID, Int32 reason, Int32 pvLoopData)
   at System.Windows.Forms.Application.ThreadContext.RunMessageLoopInner(Int32 reason, ApplicationContext context)
   at System.Windows.Forms.Application.ThreadContext.RunMessageLoop(Int32 reason, ApplicationContext context)
   at Advanced_Combat_Tracker.ActLoader.Main(String[] args)

**Comment by logicallysynced** [comment]
Investigating

**Comment by circit** [comment]
ACT also shows this error for me. I should mention that--in my case--on download, the zip file is already unblocked; no unblocking is necessary, apparently. Also, that direct installation from ACT produced the same error.

**Comment by roxaskeyheart** [comment]
@circit Can you try downloading this debug release of Chromatics 1.1.3 and see if the error persists?

http://thejourneynetwork.net/chromatics/Chromatics_debug.zip

**Comment by obcm** [comment]
I had this problem too, fixed it by installing the Chroma SDK from Razer's website. Seems like chromatics tried to load all SDKs at initialization and freaked out when it didn't find the Chroma stuff. (I have a logitech (g910) keyboard)

**Comment by roxaskeyheart** [comment]
This issue should be fixed in 1.1.3

**Comment by logicallysynced** [comment]
[Chromatics 1.1.3](https://github.com/roxaskeyheart/Chromatics/releases/tag/1.1.3) should have resolved these issues.

**Comment by obcm** [comment]
Doesn't work for me anymore ! new error.
![errot](https://cloud.githubusercontent.com/assets/16095961/11509987/590a0a2a-982f-11e5-8403-dc0bcd4713e2.jpg)

**Comment by obcm** [comment]
Plugin Initialization Failed
System.IO.FileLoadException: Could not load file or assembly 'file:///C:\Users-----\AppData\Roaming\Advanced Combat Tracker\Plugins\Chromatics\lib\CUE.NET.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
File name: 'file:///C:\Users-----\AppData\Roaming\Advanced Combat Tracker\Plugins\Chromatics\lib\CUE.NET.dll' ---> System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information.
   at Chromatics.Chromatics.InitializeDevices()
   at Chromatics.Chromatics.InitPlugin(TabPage pluginScreenSpace, Label pluginStatusText)
   at Advanced_Combat_Tracker.FormActMain.pluginPanelEnabledChecked(Object sender, EventArgs e)

   at Advanced_Combat_Tracker.FormActMain.WriteExceptionLog(Exception ex, String MoreInfo)
   at Advanced_Combat_Tracker.FormActMain.pluginPanelEnabledChecked(Object sender, EventArgs e)
   at System.Windows.Forms.CheckBox.OnCheckedChanged(EventArgs e)
   at System.Windows.Forms.CheckBox.set_CheckState(CheckState value)
   at System.Windows.Forms.CheckBox.set_Checked(Boolean value)
   at Advanced_Combat_Tracker.FormActMain.btnAddPluginPanel_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
   at System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG& msg)
   at System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG& msg)
   at System.Windows.Forms.Application.ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr dwComponentID, Int32 reason, Int32 pvLoopData)
   at System.Windows.Forms.Application.ThreadContext.RunMessageLoopInner(Int32 reason, ApplicationContext context)
   at System.Windows.Forms.Application.ThreadContext.RunMessageLoop(Int32 reason, ApplicationContext context)
   at Advanced_Combat_Tracker.ActLoader.Main(String[] args)

---

**Comment by arlarzelere3** [comment]
I can enable the plugin but my devices don't change color.  I'll create a new issue once I do some digging.
